apps/contracts/README.md

```diff
@@ -7,51 +7,50 @@
 
 ---
 
-## Install
+### Install / Compile
 
-Install the dependencies:
+`Contracts` in a NX app, and installation/building can be done by running respective commands in the project root.
 
 ```bash
-cd contracts
-npm install
-```
-
-### Compile
-
-Compile the smart contracts with Hardhat:
-
-```bash
-npm run compile
+# in project root
+yarn
+npm yarn build
```

Comment:
`npm yarn build` -> `nx run contracts:build` (?)

Reply:
Yea, fixed this as well. I mean't to write`yarn build:all` earlier. I have added commands for building separately and all of them together.